etc/adminhtml/system/adyen_moto_settings.xml

```diff
@@ -12,25 +12,25 @@
 -->
 <include xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Config:etc/system_include.xsd">
     <group id="adyen_moto_advanced_settings" translate="label" showInDefault="1" showInWebsite="1" showInStore="1" sortOrder="600">
-        <label>MOTO settings</label>
+        <label>MOTO</label>
```

Comment:
Can we add a `<comment>` to the config section like in PBL? Maybe with the info from the tooltip